ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -219,16 +219,54 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("then") {
-			pending("should start the subsequent signal after the completion of the original") {
+			it("should start the subsequent producer after the completion of the original") {
+				let (original, sink) = SignalProducer<Int, NoError>.buffer()
+
+				var subsequentStarted = false
+				let subsequent = SignalProducer<Int, NoError> { observer, _ in
+					subsequentStarted = true
+				}
+
+				let producer = original |> then(subsequent)
+				producer.start()
+				expect(subsequentStarted).to(beFalse())
```

Comment:
Can you please use `beFalsy` and `beTruthy` for greater backwards- and cross-compatibility?